#Object Information
string label
string[] all_label
#bbox Information
int32[] bbox_xmin
int32[] bbox_ymin
int32[] bbox_xmax
int32[] bbox_ymax
float32[] score
